vgrep.go

```diff
@@ -33,6 +33,7 @@ type cliArgs struct {
 	Debug       bool   `short:"d" long:"debug" description:"Verbose debug logging"`
 	Interactive bool   `long:"interactive" description:"Enter interactive shell"`
 	NoGit       bool   `long:"no-git" description:"Use grep instead of git-grep"`
+	NoRipgrep   bool   `long:"no-ripgrep" description:"Don't use ripgrep"`
```

Comment:
To be consistent with the NoHeader wording:
```suggestion
	NoRipgrep   bool   `long:"no-ripgrep" description:"Do not use ripgrep"`
```